tools/report-converter/tests/unit/smatch_output_test_files/sample.out

```diff
@@ -1 +1 @@
-files/sample.c:4 check_sample() warn: passing zero to 'ERR_PTR'
+files/sample.c:4 check_sample() [smatch.check_zero_to_err_ptr]] warn: passing zero to 'ERR_PTR'
```

Comment:
Is this just a copy and paste problem here or does smatch really output two square brackets ']]'?

Reply:
This is a copy paste error :sweat_smile: Let me correct that and the corresponding tests